### PR TITLE
Add admin process links to hcb code pages

### PR DESCRIPTION
**Pull request by malted**
Checks:
image

ACHs:
image

PayPal:
image

Disbursements:
image

### CODE REVIEW

File: app/views/hcb_codes/_ach_transfer.html.erb [hunk 1]
```diff
@@ -126,6 +126,10 @@
           <%= cancel_button %>
         </div>
       <% end %>
+
+      <% admin_tool do %>
+        <%= button_to "Go to processing", ach_start_approval_admin_path(@hcb_code.ach_transfer), class: "btn", method: :get %>
+      <% end %>
```
Comment (sampoder) [inline app/views/hcb_codes/_ach_transfer.html.erb]: Sorry this is pedantic, everywhere else we just say "Process"
```suggestion
      <% admin_tool do %>
        <%= button_to "Process", ach_start_approval_admin_path(@hcb_code.ach_transfer), class: "btn", method: :get %>
      <% end %>
```

File: app/views/hcb_codes/_disbursement.html.erb [hunk 2]
```diff
@@ -115,7 +115,13 @@
   </section>
 
   <% if @event.can_front_balance? %>
-    <section class="card__banner card__darker secondary border-top">
+    <section class="card__banner card__darker secondary border-top flex items-center g2">
+      <% if @hcb_code.disbursement.may_mark_rejected? %>
+        <% admin_tool "w-fit" do %>
+          <%= button_to "Go to processing", disbursement_process_admin_path(@hcb_code.disbursement), class: "btn", method: :get %>
```
Comment (garyhtou) [inline app/views/hcb_codes/_disbursement.html.erb]:
```suggestion
          <%= button_to "Process", disbursement_process_admin_path(@hcb_code.disbursement), class: "btn", method: :get %>
```

File: app/views/hcb_codes/_increase_check.html.erb [hunk 3]
```diff
@@ -119,6 +119,10 @@
           <%= cancel_button %>
         </div>
       <% end %>
+
+      <% admin_tool do %>
+        <%= button_to "Go to processing", increase_check_process_admin_path(@hcb_code.increase_check), class: "btn", method: :get %>
```
Comment (garyhtou) [inline app/views/hcb_codes/_increase_check.html.erb]:
```suggestion
        <%= button_to "Process", increase_check_process_admin_path(@hcb_code.increase_check), class: "btn", method: :get %>
```

File: app/views/hcb_codes/_paypal_transfer.html.erb [hunk 4]
```diff
@@ -52,6 +52,14 @@
       <%= render "hcb_codes/tags", hcb_code: @hcb_code, event: (@event || @hcb_code.event) %>
     </section>
   </turbo-frame>
+
+  <% if admin_signed_in? && @hcb_code.paypal_transfer.may_mark_rejected? %>
+    <div class="card__banner card__darker border-top overflow-hidden">
+      <% admin_tool "w-fit" do %>
+        <%= button_to "Go to processing", paypal_transfer_process_admin_path(@hcb_code.paypal_transfer), class: "btn", method: :get %>
```
Comment (garyhtou) [inline app/views/hcb_codes/_paypal_transfer.html.erb]:
```suggestion
        <%= button_to "Process", paypal_transfer_process_admin_path(@hcb_code.paypal_transfer), class: "btn", method: :get %>
```

**Review by garyhtou — APPROVED** [review]

**Pull request merged**